RiotSwiftUI/Modules/UserSessions/UserOtherSessions/Test/Unit/UserOtherSessionsViewModelTests.swift

```diff
@@ -87,10 +87,15 @@ class UserOtherSessionsViewModelTests: XCTestCase {
     
     func test_whenModelCreated_withUnverifiedFilter_viewStateIsCorrect() {
         let sessionInfos = [createUserSessionInfo(sessionId: "session 1"),
-                            createUserSessionInfo(sessionId: "session 2")]
+                            createUserSessionInfo(sessionId: "session 2", verificationState: .permanentlyUnverified),
+                            createUserSessionInfo(sessionId: "session 3", verificationState: .unknown)]
         let sut = createSUT(sessionInfos: sessionInfos, filter: .unverified)
         
-        let expectedItems = sessionInfos.filter { !$0.isCurrent }.asViewData()
+        let expectedItems = sessionInfos
+            .filter {
+                !$0.isCurrent && ($0.verificationState == .unverified || $0.verificationState == .permanentlyUnverified)
```

Comment:
would give a name to the $0 closure var a this point, when it's a single inline fast closure is fine, but a whole line with multiple uses, at this point would be more readable to name it something like sessionInfo.
Or maybe to have this even more compact and easier to read I would follow the other suggestion.